src/providers/completion.ts

```diff
@@ -62,18 +69,11 @@ async function provideCompletionItems(
     ).catch((err) => {
       console.log(`Failed to get results from completions provider '${provider}'`, err);
     });
-
+    console.log('completions', provider, completions);
+    console.log('results before merge', provider, results);
```

Comment:
I'm thinking we want to clean these and some other log statements up.

Reply:
I think I removed these particular logs the other day.